dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/features/NetworkApiLiveTest.java

```diff
@@ -42,114 +42,134 @@
-      PagedIterable<FirewallRule> firewallRulesList = api().listFirewallRules(networkDomainId);
-      assertFalse(firewallRulesList.isEmpty());
-      assertEquals(firewallRulesList.last().get().first().get().networkDomainId(), networkDomainId);
-   }
-
-   @Test(dependsOnMethods = "testListFirewallRules")
-   public void testDeleteFirewallRule() {
-      api().deleteFirewallRule(firewallRuleId);
-      FirewallRule rule = findById(api().listFirewallRules(networkDomainId).concat().toList(), firewallRuleId);
-      assertNull(rule);
-   }
-
-   @Test(dependsOnMethods = "testDeployNetworkDomain")
-   public void testDeployVlan() {
-      vlanId = api().deployVlan(networkDomainId, NetworkApiLiveTest.class.getSimpleName() + new Date().getTime(),
-            NetworkApiLiveTest.class.getSimpleName() + new Date().getTime(), DEFAULT_PRIVATE_IPV4_BASE_ADDRESS,
-            DEFAULT_PRIVATE_IPV4_PREFIX_SIZE);
-      assertNotNull(vlanId);
-      assertTrue(vlanNormalPredicate.apply(vlanId), "vlan is not in a NORMAL state after timeout");
-   }
-
-   @Test
-   public void testDeployNetworkDomain() {
-      networkDomainName = NetworkApiLiveTest.class.getSimpleName() + new Date().getTime();
-      networkDomainId = api().deployNetworkDomain(datacenters.iterator().next(), networkDomainName,
-            NetworkApiLiveTest.class.getSimpleName() + new Date().getTime() + "description", "ESSENTIALS");
-      assertNotNull(networkDomainId);
-      assertTrue(networkDomainNormalPredicate.apply(networkDomainId),
-            "network domain is not in a NORMAL state after timeout");
-   }
-
-   @Test(expectedExceptions = ResourceAlreadyExistsException.class)
-   public void testDeploySameNetworkDomain() {
-      api().deployNetworkDomain(datacenters.iterator().next(), networkDomainName, networkDomainName, "ESSENTIALS");
-   }
-
-   @AfterClass
-   public void tearDown() {
-      if (!firewallRuleIds.isEmpty()) {
-         for (String firewallRuleId : firewallRuleIds) {
-            api().deleteFirewallRule(firewallRuleId);
-         }
-      }
-      if (portListId != null) {
-         api().deletePortList(portListId);
-      }
-      if (vlanId != null) {
-         api().deleteVlan(vlanId);
-         assertTrue(vlanDeletedPredicate.apply(vlanId), "vlan is not in a DELETED state after timeout");
-      }
-      if (networkDomainId != null) {
-         api().deleteNetworkDomain(networkDomainId);
-         assertTrue(networkDomainDeletedPredicate.apply(networkDomainId),
-               "network domain is not in a DELETED state after timeout");
-      }
-   }
-
-   private NetworkApi api() {
-      return api.getNetworkApi();
-   }
-
-   private FirewallRule findById(List<FirewallRule> collection, String id) {
-      for (FirewallRule rule : collection) {
-         if (rule.id().equals(id)) {
-            return rule;
-         }
-      }
-      return null;
-   }
+public class NetworkApiLiveTest extends BaseDimensionDataCloudControlApiLiveTest
```

Comment:
Can you please reformat this file according to the jclouds style guidelines?

Reply:
Thanks for looking @nacx , I put a new image on my machine recently so had to set everything up again. Forgot to apply the formatting settings.